services/web/server/src/simcore_service_webserver/resource_manager/garbage_collector.py

```diff
@@ -42,32 +44,37 @@
 
 from .config import (
     APP_CLIENT_REDIS_LOCK_KEY,
-    APP_GARBAGE_COLLECTOR_KEY,
     GUEST_USER_RC_LOCK_FORMAT,
     get_garbage_collector_interval,
 )
 from .registry import RedisResourceRegistry, get_registry
 
 logger = logging.getLogger(__name__)
+database_errors = (psycopg2.DatabaseError, asyncpg.exceptions.PostgresError)
```

Comment:
Could you also consider renaming `database_errors ` to `GroupedDatabaseErrors` ?

Reply:
it is not a class therefore it has to be snake-case.